system/hubcopter-seeds/templates/seed.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: "openstack.stable.sap.cc/v1"
+kind: OpenstackSeed
+metadata:
+  name: hubcopter-seed
+
+spec:
+  requires:
+    - {{ .Values.global.keystoneNamespace }}/keystone-seed
```

Comment:
``[ERROR] templates/: template: hubcopter/templates/seed.yaml:8:16: executing "hubcopter/templates/seed.yaml" at <.Values.global.keystoneNamespace>: nil pointer evaluating interface {}.keystoneNamespace``

Not sure what the problem is here

Reply:
This value usually comes from the `globals.yaml` values file. We don't have that one during `helm lint`, so you need to supply the respective values in `ci/test-values.yaml`.